csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -227,6 +227,52 @@ public void CanCreateAndDisposeSessionWithModelPath()
             }
         }
 
+        [Fact]
+        private void validateProviderOptions()
+        {
+            string modelPath = Path.Combine(Directory.GetCurrentDirectory(), "squeezenet.onnx");
+
+#if USE_TENSORRT
```

Comment:
should this entire test be under #if USE_TENSORRT macro?
will there be build warning for non tensorrt builds due to unused variable?

Reply:
Moved this macro to cover entire test case.